etc/databases/world/updates/updates.sql

```diff
@@ -30510,6 +30510,47 @@ begin not atomic
         insert into applied_updates values ('080320231');
     end if;
 
+    -- 10/03/2023 1
+    if (select count(*) from applied_updates where id='100320231') = 0 then
+        -- Despawn herbalism nodes not in 0.5.3 alpha
+        -- Dreamfoil
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176584;
+        -- Black Lotus
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176589;
+        -- Golden Sansam
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176583;
+        -- Gromsblood
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 142145;
+        -- Plaguebloom
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176587;
+        -- Sungrass
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 142142;
+        -- Mountain Silversage
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176586;
+
+        -- Despawn mining nodes not in 0.5.3 alpha
+        -- Thorium (Rich, Small, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(177388, 123848, 175404, 324, 150082, 176643);
+        -- Truesilver (Deposit, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(123309, 2047, 150081);
+        -- Mithril (Deposit, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(2040, 150079, 176645, 123310);
```

Comment:
Mithril do exist in 0.5.3, I'd not remove the deposits.

Reply:
The deposits aren't properly itemized though, they only contain ore and a gem. They also can't be mined.

Reply:
Please leave them until / if we find more proof. Both Mithril Ore and Bars have a very low item entry. We need to take a very conservative approach when despawning things. This applies to the chests too, I'd only despawn Winterspring ones (by spawn_id) since we are 100% sure that area was barely even started in 0.5.3.

Reply:
Burning Steppes too, the place is completely empty and Blackrock Mountain has a hole in it where the entrance will be.

Reply:
Makes sense.

Reply:
I'll keep this to ore/herbs for now, sans Mithril.